server/src/main/java/org/apache/druid/client/selector/AbstractTierSelectorStrategy.java

```diff
@@ -34,7 +34,7 @@
  */
 public abstract class AbstractTierSelectorStrategy implements TierSelectorStrategy
 {
-  private final ServerSelectorStrategy serverSelectorStrategy;
+  protected final ServerSelectorStrategy serverSelectorStrategy;
```

Comment:
I think this can be restored to private now that PreferredTierSelectorStrategy is delegating to highest or lowest priority strategy instead of calling server selector direct